utils/streamlit_utils.py

```diff
@@ -34,236 +40,256 @@
 pd.options.display.float_format = "{:,.3f}".format # '{:20,.2f}'.format
 
 def sidebar_header():
+
+    side_bar_text = '<p style="font-size: 16px;font-weight:bold;">To use this tool, follow the steps below👇</p>'
+    st.sidebar.markdown(
+    side_bar_text,
+    unsafe_allow_html=True,
+   )
+   #return 0
+
+
+
+    '''
     st.sidebar.markdown("""This demo showcases the 
```

Comment:
This stuff is in bold, which makes it looks not as good as the design in the figma: With everything in bold, using bold doesn't help anything to stick out and instead makes it seem like everything is a bit more intense. Can you change the writing style to match the design here: https://www.figma.com/file/xvBWSyzuURvtIaMKAjAPZY/Data-measurements-tool?node-id=0%3A1 ?